magpie/adapter/magpieowssecurity.py

```diff
@@ -82,17 +82,16 @@ def update_request_cookies(self, request):
             headers.update({"Homepage-Route": "/session", "Accept": CONTENT_TYPE_JSON})
             session_resp = requests.get(magpie_auth, headers=headers, verify=self.twitcher_ssl_verify)
             if session_resp.status_code != HTTPOk.code:
-                raise OWSAccessForbidden("Not authorized to access this resource. " +
+                raise OWSAccessForbidden("Not authorized to access this resource. "
                                          "Provider login failed with following reason: [{}]."
                                          .format(session_resp.reason))
 
             # use specific domain to differentiate between `.{hostname}` and `{hostname}` variations if applicable
-            # noinspection PyProtectedMember
-            request_cookies = session_resp.request._cookies
+            request_cookies = session_resp.request._cookies  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _cookies of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)